mne/encoding/feature.py

```diff
@@ -0,0 +1,269 @@
+"""Classes for transforming, preprocessing, and fitting encoding models."""
+# Authors: Chris Holdgraf <choldgraf@gmail.com>
+#          Jona Sassenhagen <jona.sassenhagen@gmail.de>
+#          Jean-Remi King <jeanremi.king@gmail.com>
+#
+# License: BSD (3-clause)
+import numpy as np
+from scipy.sparse import spmatrix, csr_matrix
+from ..utils import warn
+
+
+class FeatureDelayer(object):
```

Comment:
don't really like name. Maybe TimeDelayEmbedding or better?
also it should inherit from BaseEstimator.